brave-core/src/test/java/com/github/kristofa/brave/LocalTracerTest.java

```diff
@@ -82,6 +91,38 @@ public void startNewSpan() {
     }
 
     /**
+     * When a local span is started without a timestamp, microseconds and a tick are recorded for
```

Comment:
Sure. But there was nothing in this apidoc that was specific to either test case. hence the re-use looks appropriate.